modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -236,7 +236,7 @@ For an example, see <<import-tool-multiple-input-files-regex-example>>.
 |
 
 |--normalize-types[=true\|false]
-|Whether or not to normalize property types to Cypher types, e.g. `int` becomes `long` and `float` becomes `double`.
+|When true, non-array property values will be converted to their equivalent Cypher types. For example, all integer values will be converted to 64-bit long integers. This takes up more space on disk, but avoids Cypher converting the type during queries.
```

Comment:
@renetapopova do you know if these are pulled automatic from [our codebase](https://github.com/neo-technology/neo4j/blob/dbefc055901079cd05287fd8d99f1d7c9e67764c/public/community/import-tool/src/main/java/org/neo4j/importer/ImportCommand.java#L338)?

Reply:
No, they are not. That's why @mauvo raised this PR here. However, we have a test that validates the neo4j-admin options. I added a comment a bit further above that we need to update the codebase as well when we agree on the descriptions.